File: 04-Kuku/Answers/Python/kuku2-ans.py
#!usr/bin/python
# -*- coding: UTF-8 -*-

# kuku2-ans.py
# By kota ninomiya

def create_kuku_table(number:int):
    """九九の表を作成する関数
    
    Args:
        number (int): 上限となる数字
    """
    message = ''
    for i in range(1, number): 
        for j in range(1, number):
            message += '{}*{}='.format(i, j)
            if i*j < 10: message += ' '
            message += '{}, '.format(i*j)
        message = message.rstrip(', ')
        message += '\n'

    print(message)


def main():
    create_kuku_table(10)
        


if __name__ == '__main__':
    main()

File: 04-Kuku/Answers/JavaScript/kuku2-ans.js
// kuku2-ans.js
// By kota ninomiya

/**
 * 九九の表を作る関数
 * @param {Number} number 上限となる数字
 */
function createKukuTable(number){
    message = ''
    for(let i=1; i<number; i++){
        for(let j=1; j<number; j++){
            message += `${i}*${j}=${i*j<10 ? ' ' : ''}${i*j}, `
        }
        message = message.slice(0, -2)
        message += `\n`
    }
    console.log(message)
}

createKukuTable(5)